### ISSUE TITLE
WhatsApp will block you if you send too many requests

### ISSUE THREAD

**Issue by Shalom-Borohov** [adiwajshing/Baileys#1946]
I was tinkering with the project and wanted to get a bunch of information, it was working like a charm but then WhatsApp blocked my account for violation of their terms and conditions. 

Maybe its worth mentioning in the Readme note section that this could happen for other people that follow.

**Comment by adhkarbot** [comment]
+1

**Comment by flasd** [comment]
How many messages did you send? and how fast did you send them?

**Comment by josuetex** [comment]
> How many messages did you send? and how fast did you send them?

+1

**Comment by Alhar6i** [comment]
> I was tinkering with the project and wanted to get a bunch of information, it was working like a charm but then WhatsApp blocked my account for violation of their terms and conditions.
> 
> Maybe its worth mentioning in the Readme note section that this could happen for other people that follow.

Have you used your account as a normal WhatsApp account before using the Baileys?
Whatsapp AI mechanism is smart enough to detect abnormal behavior of the WA accounts especially if they are new accounts.

**Comment by Shalom-Borohov** [comment]
> How many messages did you send? and how fast did you send them?

I sent requests and wrote the responds to a file in batches of 10000 requests per file save, It ran for about 15 minutes, if I remember correctly, before it logged me out and banned my account.

**Comment by Shalom-Borohov** [comment]
> > I was tinkering with the project and wanted to get a bunch of information, it was working like a charm but then WhatsApp blocked my account for violation of their terms and conditions.
> > Maybe its worth mentioning in the Readme note section that this could happen for other people that follow.
> 
> Have you used your account as a normal WhatsApp account before using the Baileys? Whatsapp AI mechanism is smart enough to detect abnormal behavior of the WA accounts especially if they are new accounts.

Yes. 
I tried again today with a fake account I created an hour ago and it banned me after a minute of batch request sending.
I guess I'll try sending the requests with timeouts between them.

**Comment by mniiinm** [comment]
This problem is not related to this library. If you register with a fake account and after that, send bulk messages through the WhatsApp application itself, your account will be banned.

**Comment by Shalom-Borohov** [comment]
> This problem is not related to this library. If you register with a fake account and after that, send bulk messages through the WhatsApp application itself, your account will be banned.

I understand that, I think it's worth mentioning in the Readme note section that this sort of thing could happen because I initially used my personal account and after I got banned, I registered a new fake account.

I would have thought twice whether I should do it if that information was written in the Readme note section, and I definitely wouldn't have used my personal account for that.

Eventually, no harm done, WhatsApp unbanned me after a few (critical) hours, but that gave me a scare.
